packages/side-nav/README.md

```diff
@@ -18,174 +18,367 @@ yarn add @leafygreen-ui/side-nav
 npm install @leafygreen-ui/side-nav
 ```
 
-## Peer Dependencies
-
-| Package                              | Version  |
-| ------------------------------------ | -------- |
-| `@leafygreen-ui/leafygreen-provider` | `^1.1.0` |
```

Comment:
Is this no longer a peer dependency?

Reply:
I didn't even notice we had this. Is it worth having? A lot of packages will transitively require it too because they use the Button, Interaction-ring, useUsingKeyboardContext, etc. I definitely failed to add this section to the Select and Copyable readmes

Reply:
It's _probably_ something we should keep around, but missing it in a component or two isn't going to make a huge impact since it's likely everywhere already has this component installed.

@brookescarlett I don't feel strongly about this. You wanna make this call?

Reply:
I think we tried to say that it's a peer dep of every component in the repo. I don't feel strongly one way or another, but do think that we should be consistent.